spec/rubocop/cop/rspec/repeated_example_spec.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+describe RuboCop::Cop::RSpec::RepeatedExample do
+  subject(:cop) { described_class.new }
+
+  it 'registers an offense for repeated example' do
+    expect_violation(<<-RUBY)
+      describe 'doing x' do
+        it "does x" do
+                    ^^ Don't repeat examples within an example group.
```

Comment:
Yeah. I need to make the `ExpectViolation` thing support this. RuboCop currently will highlight the whole expression but only display the first line in its output. The `ExpectViolation` thing doesn't know how to handle multiple lines yet